### PR TITLE
feat(examples): candy machine code examples

### LINKED ISSUE #473: Metaplex JS SDK Candy Machine examples
Hey! Metaplex JS SDK now supports creating Candy Machine v2 via code in [v0.17.0](https://github.com/metaplex-foundation/js/releases/tag/js%2Fv0.17.0). I would like to add code examples regarding the Candy Machine module under the[ NFTs reference section](https://solanacookbook.com/references/nfts.html)

### PR DESCRIPTION
**Pull request by 0xMukesh**
- Closes #473

### CODE REVIEW

**Comment by 0xMukesh** [comment]
Weird, I can't see the "Candy Machine v2" header under the NFTs reference section

| [Vercel preview](https://solana-cookbook-git-fork-0xmukesh-feat-candy-d80f9e-solana-labs.vercel.app/references/nfts.html) | Local development environment |
| -------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------- |
| image  | image

File: docs/references/nfts.md [hunk 1]
```diff
@@ -235,3 +235,183 @@ This can all be done using [`findDataByOwner`](https://github.com/metaplex-found
...
+
+@[code](@/code/nfts/candy-machine/find-via-address.preview.en.ts)
+
+  </template>
+
+  </SolanaCodeGroupItem>
+
+</SolanaCodeGroup>
+
+### How to find minted NFTs
```
Comment (jacobcreech) [inline docs/references/nfts.md:354]:
```suggestion
### How to find minted NFTs from a Candy Machine
```

File: docs/references/nfts.md [hunk 2]
```diff
@@ -235,3 +235,183 @@ This can all be done using [`findDataByOwner`](https://github.com/metaplex-found
...
+
+@[code](@/code/nfts/candy-machine/find-minted-nfts.preview.en.ts)
+
+  </template>
+
+  </SolanaCodeGroupItem>
+
+</SolanaCodeGroup>
+
+### How to insert items
```
Comment (jacobcreech) [inline docs/references/nfts.md:375]:
```suggestion
### How to insert items into a Candy Machine
```

File: docs/references/nfts.md [hunk 3]
```diff
@@ -235,3 +235,183 @@ This can all be done using [`findDataByOwner`](https://github.com/metaplex-found
...
+
+@[code](@/code/nfts/candy-machine/insert-items.preview.en.ts)
+
+  </template>
+
+  </SolanaCodeGroupItem>
+
+</SolanaCodeGroup>
+
+### How to mint an NFT
```
Comment (jacobcreech) [inline docs/references/nfts.md:396]:
```suggestion
### How to mint an NFT from a Candy Machine
```

**Review by jacobcreech — CHANGES_REQUESTED** [review]
Awesome work! Just a few requested changes to help improve SEO

**Review by jacobcreech — APPROVED** [review]

**Comment by 0xMukesh** [comment]
Hey @jacobcreech, thank you for merging the PR. Have checked the [production site](https://solanacookbook.com) and it can't see the "Candy Machine v2" header under the NFTs reference section. Did I miss something to change in vuepress's configuration or anything else?

**Comment by jacobcreech** [comment]
@0xMukesh Unfortunately I temporarily disabled the build because vercel can't handle building so many pages of vuepress at once. Going to have to manually build and push until I get a better CICD pipeline

**Pull request merged**